Code/GraphMol/RGroupDecomposition/RGroupDecomp.h

```diff
@@ -143,6 +143,19 @@ class RGroupDecomposition {
   //! return rgroups in column order group[attachment_point][row] = ROMol
   RGroupColumns getRGroupsAsColumns() const;
 };
+
+unsigned int RGroupDecompose(RGroupRows &rows,
```

Comment:
Minor thing: in the interest of consistency with the rest of the API (or at least most of it), the argument used to return results  - `rows` here, `cols` below - should be the last non-optional argument. So this function would be `RGroupDecompose(cores,mols,rows,unmatched,options)`